oauth2-useragent/pom.xml

```diff
@@ -115,6 +131,15 @@
             </configuration>
           </plugin>
           <plugin>
+            <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
Short answer: It's the only one that matters, yes.

Long answer:
The project is split up into modules to be able to selectively enforce Java 6 for the `core` project and Java 7 for the `javafx` project, which also requires a compile-time reference to the JavaFX JARs (but we don't want to enforce JavaFX be there at runtime).

Once we have SWT, we'll need a compile-time reference to the SWT JARs and then the SWT provider may download those on-demand (or the consumer of this library will ship with them).  Again, we don't want to necessarily have a dependency on SWT.

So, we "cheat" by having these things be separate Maven modules during development and then we aggregate them for packaging as if they had been in a single module the whole time.  This results in a single JAR, a single -sources JAR and a single -javadoc JAR.